Dockerfile.allSdks

```diff
@@ -48,5 +45,5 @@ ENV PATH=${PATH}:/usr/src/firebolt/mock-firebolt/cli
 WORKDIR /usr/src/firebolt/mock-firebolt/server
 
 ENTRYPOINT [ "npm",  "run", "start" ]
-# By default, only the core SDK is enabled. When using 'docker run', users can pass --manage --discovery as command-line parameters
+# By default, only the mono-sdk retrieved from HTTP is enabled. When using 'docker run', users can pass --discovery as command-line parameters
```

Comment:
I think the mono-sdk is still just being referred to as "core". Our documentation should match that. Can we confirm what the mono SDK is being called?

Reply:
Checking with Jeremy on the same

Reply:
confirmed the correct reference is core and updated the same